packages/vscode-extension/src/utilities/openFileAtPosition.ts

```diff
@@ -12,18 +12,22 @@ export async function openFileAtPosition(
   });
 
   const selection = new Range(line0Based, column0Based, line0Based, column0Based);
+  const activeRNIDEColumn = window.tabGroups.all.find(
+    (group) => group.activeTab?.label === "React Native IDE"
+  )?.viewColumn;
+  const column = activeRNIDEColumn === ViewColumn.One ? ViewColumn.Beside : ViewColumn.One;
```

Comment:
Documentation says: "A symbolic editor column representing the column to the side of the active one." but that side seems to always be on the right of Radon so it doesn't work with two tabs setup where Radon is on right and code on left, nor vertical two tabs layout, because `Beside` in those cases opens tab in new column.